From b676ab1e1c22d5294f0e743cee3ef9e72aff4518 Mon Sep 17 00:00:00 2001
From: Mingliang Liu <liuml07@apache.org>
Date: Wed, 5 Jul 2017 11:10:57 -0700
Subject: [PATCH 2558/2748] HDFS-12089. Fix ambiguous NN retry log message in
 WebHDFS. Contributed by Eric Badger

(cherry picked from commit 6436768baf1b2ac05f6786edcd76fd3a66c03eaa)
(cherry picked from commit db6e8ff7479780fdbd6199e826facbfc648defb6)

Conflicts:
	hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java

Change-Id: I0b09e327ca004f7f7a27f757b9699332cf5ae1e0
---
 .../apache/hadoop/hdfs/web/WebHdfsFileSystem.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java
index 396d40e..ecf1b2b 100644
--- a/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java
+++ b/hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/web/WebHdfsFileSystem.java
@@ -719,7 +719,7 @@ private void shouldRetry(final IOException ioe, final int retry
 
           if (isRetry || isFailoverAndRetry) {
             LOG.info("Retrying connect to namenode: " + nnAddr
-                + ". Already tried " + retry + " time(s); retry policy is "
+                + ". Already retried " + retry + " time(s); retry policy is "
                 + retryPolicy + ", delay " + a.delayMillis + "ms.");
 
             if (isFailoverAndRetry) {
-- 
1.7.9.5

